sa/database.go

```diff
@@ -12,11 +13,34 @@ import (
 	blog "github.com/letsencrypt/boulder/log"
 )
 
+// DbSettings contains tuning setting for the sql driver. Optional
+// fields should only be applied if their zero value matches the default
+// of the sql driver. SA is the authoritative closer of connections when
```

Comment:
It's not clear what "should" means in this context. Is that imposing a recommendation on the caller, or a recommendation to people who edit DbSettings in the future? Or an expression of uncertainty?

I think what you want to say here is "the zero value for each field means 'use the default setting from database/sql'".